src/Stryker.Core/Stryker.Core/Instrumentation/DefaultInitializationEngine.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Helpers;
+
+namespace Stryker.Core.Instrumentation
+{
+    internal class DefaultInitializationEngine : BaseEngine<BaseMethodDeclarationSyntax>
```

Comment:
This engine can be used to place default initializers on method scope variables, correct? Could you add a small docstring to this class explaining the intent.

Reply:
yes. I added documentation to every engine for clarification.

Reply:
Awesome :)